snap/snapcraft.yaml

```diff
@@ -780,51 +763,50 @@ parts:
+  # device-virtual, however we currently don't publish an independent
+  # snap of device-virtual. Maybe we should, as this would simplify
+  # this snap.
   device-virtual-go:
     source: https://github.com/edgexfoundry/device-virtual-go.git
     source-depth: 1
-    source-tag: v1.2.1
+    source-tag: master
     plugin: make
     after: [go-build-helper]
     override-build: |
-      # create VERSION file (supposed to be created by jenkins pipeline...)
-      echo "1.2.1" > ./VERSION
+      echo "2.0.0-dev" > ./VERSION
       make build
 
       install -DT "./cmd/device-virtual" "$SNAPCRAFT_PART_INSTALL/bin/device-virtual"
       install -DT "./cmd/res/configuration.toml" \
         "$SNAPCRAFT_PART_INSTALL/config/device-virtual/res/configuration.toml"
+      install -DT "./cmd/res/devices/devices.toml" \
+          "$SNAPCRAFT_PART_INSTALL/config/device-virtual/res/devices/devices.toml"
+
+      mkdir -p $SNAPCRAFT_PART_INSTALL/config/device-virtual/res/profiles
 
       for profileType in bool float int uint binary; do
-        install -T "./cmd/res/device.virtual.$profileType.yaml" \
-          "$SNAPCRAFT_PART_INSTALL/config/device-virtual/res/device.virtual.$profileType.yaml"
+        install -T "./cmd/res/profiles/device.virtual.$profileType.yaml" \
+          "$SNAPCRAFT_PART_INSTALL/config/device-virtual/res/profiles/device.virtual.$profileType.yaml"
       done
 
       install -DT "./Attribution.txt" \
          "$SNAPCRAFT_PART_INSTALL/usr/share/doc/device-virtual/Attribution.txt"
       install -DT "./LICENSE" \
          "$SNAPCRAFT_PART_INSTALL/usr/share/doc/device-virtual/LICENSE"
 
+  # TODO: vet all the correct files are available...
   app-service-config:
-    source: https://github.com/edgexfoundry/app-service-configurable.git
-    # TODO(Hanoi): update to source-tag for release
-    source-branch: master
-    plugin: make
-    build-packages: [gcc, git, libzmq3-dev, pkg-config]
-    stage-packages: [libzmq5]
-    after: [go-build-helper]
-    override-build: |
-      make build
-
-      # install the service binary, configuration, and license files
-      install -DT "./app-service-configurable" \
-         "$SNAPCRAFT_PART_INSTALL/bin/app-service-configurable"
-      install -DT "./res/rules-engine/configuration.toml" \
-         "$SNAPCRAFT_PART_INSTALL/config/app-service-configurable/res/rules-engine/configuration.toml"
-      install -DT "./Attribution.txt" \
-         "$SNAPCRAFT_PART_INSTALL/usr/share/doc/app-service-configurable/Attribution.txt"
-      install -DT "./LICENSE" \
-         "$SNAPCRAFT_PART_INSTALL/usr/share/doc/app-service-configurable/LICENSE"
+    plugin: nil
+    stage-snaps:
+      - edgex-app-service-configurable/edge
+    stage:
+      - -res/functional-tests/configuration.toml
+      - -res/http-export/configuration.toml
+      - -res/mqtt-export/configuration.toml
+      - -res/push-to-core/configuration.toml
+      - -res/sample/configuration.toml
```

Comment:
What about `rules-engine` profile?

Reply:
It's installed correctly. The syntax above is actually telling snapcraft *not* to install the listed profiles.